app/src/main/java/com/cmput301f20t21/bookfriends/ui/browse/BrowseSearchFragment.java

```diff
@@ -0,0 +1,55 @@
+package com.cmput301f20t21.bookfriends.ui.browse;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.SearchView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentContainerView;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.RecyclerView;
+import com.cmput301f20t21.bookfriends.R;
+
+public class BrowseSearchFragment extends Fragment {
```

Comment:
we won't use this fragment if we are modifying the recycler adapter's list. If take a look at the `fragment_browse.xml`, notice the fragment is full width with an elevation. That means it's an "overlaying" fragment on top of everything.

As we discussed, we won't pop up an overlay on search. Instead, we will update the existing array inside the browse view model. (Right now it's still overlaying, try to set the background for the `browse_search_list_container_fragment` to white.)

To conclude, we still need the onCreateOptionsMenu, so perhaps move it to the browse fragment and remove the search container. Everything else I think is good to remove.

Reply:
How about now?